### PR TITLE
Add 'close' methods to sync::borrowed::RedisClientRemote and sync::ow…

### PR DESCRIPTION
**Pull request by kevinwilson541**
…ned::RedisClientRemote

### CODE REVIEW

File: src/sync/borrowed.rs [hunk 1]
```diff
@@ -164,6 +164,19 @@ impl RedisClientRemote {
     Box::new(commands_ft)
   }
 
+  /// Flush and close the Sender channel this instance receives messages through.
+  pub fn close(&mut self) {
+    let mut tx_guard = self.command_tx.write();
+    let mut tx_ref = tx_guard.deref_mut();
+
+    match *tx_ref {
```
Comment (aembke) [inline src/sync/borrowed.rs]: just to keep things consistent, can you do an `if let` here instead of a `match` with an empty block for the `None` variant

Comment (kevinwilson541) [reply src/sync/borrowed.rs]: For sure, I'll update the PR in a bit and let you know when it's ready.

Comment (kevinwilson541) [reply src/sync/borrowed.rs]: PR updated.

**Comment by aembke** [comment]
Fixes https://github.com/azuqua/fred.rs/issues/17

**Pull request merged**